src/main/java/io/appium/java_client/remote/MobileCapabilityType.java

```diff
@@ -115,4 +115,11 @@
      * If {@code false} then it is switched to W3C mode.
      */
     String FORCE_MJSONWP = "forceMjsonwp";
+    
+    /**
+     * ANDROID ONLY
+     * Desired capability that always accepts permissions when the app requires them when set to "true".
+     * Default value is false
+     */
+    String AUTO_GRANT_PERMISSIONS = "autoGrantPermissions";
```

Comment:
It's available in https://github.com/appium/java-client/blob/master/src/main/java/io/appium/java_client/remote/AndroidMobileCapabilityType.java#L244